src/VirtualList/VirtualList.js

```diff
@@ -20,25 +20,30 @@ class VirtualList extends React.Component {
 
     this.state = this.getVirtualState(this.props);
 
-    // Replace onScroll by debouncing
+    // Replace onScroll by throttling
     if (this.props.scrollDelay > 0) {
       this.onScroll = Util.throttle(
         this.onScroll.bind(this),
-        this.props.scrollDelay
+        this.props.scrollDelay,
+        // Fire on both leading and trailing edge to minize flash of
+        // un-rendered items
+        {leading: true, trailing: true}
```

Comment:
Wonder if this is the right solution... Seems that we actually want to make sure that the handler is called _n_ times per second when needed. Anyway this is fine for now.